db/migrate/20240106165245_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.string :title
+      t.text :text
+      t.integer :likes_counter, default: 0
+      t.integer :comments_counter, default: 0
+
+      t.timestamps
+    end
+    add_index :posts, :author_id
```

Comment:
Nice job so far👍🏼. Just as a requirement, kindly do well to add the necessary foreign keys as shown in the [ERD ](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_app_erd_v1_1.png)😉.

<img width="962" alt="image" src="https://github.com/Ochiengsteven/art-post/assets/64643995/2aac4674-e684-4c01-bfcb-e49bcd62330d">

Kindly add the necessary foreign keys for the post, likes, and comments😉.

For instance, we can make some improvements here in the post table by adding foreign keys as shown below.

```suggestion
  def change
    create_table :posts do |t|
      t.integer :author_id
      t.string :title
      t.text :text
      t.integer :likes_counter, default: 0
      t.integer :comments_counter, default: 0

      t.timestamps
    end
    add_foreign_key :posts, :users, column: :author_id
    add_index :posts, :author_id
```

Kindly note that the above is a suggestion and you should try to implement it in any way that works😉